mm2src/mm2_tests.rs

```diff
@@ -67,6 +67,8 @@ async fn enable_coins_eth_electrum(
     );
     replies.insert("ETH", enable_native(mm, "ETH", eth_urls).await);
     replies.insert("JST", enable_native(mm, "JST", eth_urls).await);
+    #[cfg(feature = "zhtlc")]
+    replies.insert("ZOMBIE", enable_native(mm, "ZOMBIE", eth_urls).await);
```

Comment:
Is it correct to specify the same urls as we do for the ETH coin?

Reply:
ETH URLs are simply ignored for native UTXO mode so it's fine :slightly_smiling_face: